test/command.js

```diff
@@ -1,10 +1,8 @@
-import path from 'node:path';
-import process from 'node:process';
-import {fileURLToPath} from 'node:url';
 import test from 'ava';
 import {execa, execaSync, execaCommand, execaCommandSync} from '../index.js';
+import {setFixtureDir} from './helpers/fixtures-dir.js';
 
-process.env.PATH = fileURLToPath(new URL('fixtures', import.meta.url)) + path.delimiter + process.env.PATH;
+setFixtureDir();
```

Comment:
How the `PATH`/`Path` environment variable in the GitHub actions Windows environment seems to have changed, making the CI tests fail. In particular, its name is now `Path` instead of `PATH`.
Using `path-key` fixes this. I put this as a shared helper to avoid code duplication.